.github/workflows/generate-code-coverage.yaml

```diff
@@ -0,0 +1,51 @@
+name: Generate Code Coverage
+on: [pull_request]
+  
+jobs:
+  cover-base:
+    name: Generate Base Coverage
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout base
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.base.ref }}
+
+      - name: Set up go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '^1.20.2'
+
+      - name: Generate report
+        run: |
+          go test -coverprofile base-coverage.tmp ./cmd/... ./pkg/...
```

Comment:
Any chance we can get the `-html=coverage.out` as an artifact and link the file in the comment?

Reply:
I don't believe so, as we would have to host it somewhere. As far as I can tell GitHub does not host artifacts for you.

I do have an idea to show the code coverage output on the prow test runs, but didn't think it was appropriate for this PR (and might not be possible? needs more investigation)